test/runnable/test15371.d

```diff
@@ -0,0 +1,31 @@
+// EXTRA_SOURCES: imports/test15371.d
+
```

Comment:
Runnable tests are _very_ expensive. Please move this to compilable and use things like static assert or pragma(msg).